protected/humhub/modules/admin/models/forms/PasswordEditForm.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace humhub\modules\admin\models\forms;
+
+use humhub\modules\user\models\Password;
+
+/**
+ * PasswordEditForm
+ * used to edit password of other users by admin
+ */
+class PasswordEditForm extends Password
+{
+    const SCENARIO_EDIT_ADMIN = 'editAdmin';
+
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        parent::init();
+        $this->scenario = self::SCENARIO_EDIT_ADMIN;
```

Comment:
Just wondering, why do we need this scenario? I don't see any other use-cases of this class.

Reply:
Ah ok  its derifed from Password model.